src/interface/report/Results/index.tsx

```diff
@@ -266,9 +281,11 @@ class Results extends React.PureComponent<Props, State> {
         const tab = results.tabs.find((tab) => tab.url === selectedTab);
 
         return (
-          <div className="container">
-            <ErrorBoundary>{tab ? tab.render() : '404 tab not found'}</ErrorBoundary>
-          </div>
+          <CombatLogParserProvider combatLogParser={parser}>
+            <div className="container">
+              <ErrorBoundary>{tab ? tab.render() : '404 tab not found'}</ErrorBoundary>
+            </div>
+          </CombatLogParserProvider>
```

Comment:
rather than wrapping each return value here, we should wrap the `renderContent` call on line 463 IMO